examples/js/api-lifecycle.test.js

```diff
@@ -0,0 +1,173 @@
+'use strict'
+
+const shared = require('./shared')
+const request = require('supertest')(shared.apiUrl)
+
+const axios = require('axios')
+const { describe, expect } = require('@jest/globals')
+const api = axios.create({
+    baseURL: shared.apiUrl,
+    timeout: 1000000000
+})
+
+describe('Lifecycle', () => {
+
+    console.log('Running lifecycle tests')
+    console.log('Environment:', process.env.NODE_ENV)
+
+    // Ensuring all the env vars are set
+    shared.checkEnvVars()
+
+    const apiBaseUrl = shared.apiUrl + '/api/v1'
+    // Before using API endpoints, obtaining an access token with the correct scopes is necessary, 
+    // as the token's scopes must align with those required by the endpoint. 
+    // Although tokens can be reused within their validity period, 
+    // it's recommended to request new tokens for different scopes as needed, 
+    // prioritizing minimal scope use for security. 
+    // The guidance assumes tokens won't expire quickly for simplicity, 
+    // but in practice, handling token expiration by refreshing or acquiring new tokens 
+    // for each request is essential for continuous and secure API access.
```

Comment:
I might remove this and instead comment the places where we request different scopes, explaining, for example, that patient endpoints require the `dockhealth/patient.all.read` or `dockhealth/patient.all.write` scope. Also, I think I've changed my mind about scopes -- I think it's fine to request all the scopes at once. If I were writing this example, I would just request them all -- it's much more performant, and probably not any riskier.